packages/cxl-ui/src/components/cxl-course-card.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import cxlCourseCardStyles from '../styles/cxl-course-card-css.js';
+
+@customElement('cxl-course-card')
+export class CXLCourseCardElement extends LitElement {
+  static get styles() {
+    return [cxlCourseCardStyles];
+  }
+
+  separator = html`<span> | </span>`;
+
+  @state() _hasSlottedHeaderTags = false;
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) type = 'Course';
+
+  @property({ type: String }) title = '';
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: String, attribute: 'avatar-image' }) avatarImage = '';
+
+  @property({ type: Boolean, reflect: true }) new = false;
+
+  @property({ type: String, attribute: 'cta-label' }) ctaLabel = 'View';
+
+  @property({ type: Boolean, attribute: 'cta-url' }) ctaUrl = false;
+
+  @property({ type: String, attribute: 'more' }) more = '';
+
+  _tagsSlotChanged (e) {
+    const slot = e.target
+    const children = slot.assignedNodes()
+    this._hasSlottedHeaderTags = !!children.length
+  }
+
+  render() {
+    return html`
+      <!-- Generic Slot, kept for backwards compatibility with current usage of theme="cxl-testimonial" and any other slotted content targeted by global styles -->
+      <slot></slot>
```

Comment:
It is new component now. Is it still needed as before?

Reply:
No, you're right, taking it out